hardware/electrical/power_supply.kicad_sch

```diff
@@ -5,6 +5,2544 @@
+    (uuid 848b1429-22cf-4f79-8763-24032a1ecd33)
+    (property "Reference" "C2" (id 0) (at 133.35 55.88 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Value" "0.1uF" (id 1) (at 133.35 58.42 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Footprint" "Capacitor_SMD:C_0402_1005Metric" (id 2) (at 130.5052 53.34 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 129.54 57.15 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 006aa44e-bcdb-4b27-8611-0e9c666e05de))
+    (pin "2" (uuid 64d993a5-9b4c-4e76-a727-3cb1983c8ba0))
+  )
+
+  (symbol (lib_id "Device:C") (at 76.2 60.96 0) (mirror x) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 868e3ab4-7549-4ad2-b897-c37a417778bf)
+    (property "Reference" "C1" (id 0) (at 80.01 59.69 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Value" "4.7uF" (id 1) (at 80.01 62.23 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Footprint" "Capacitor_SMD:C_0402_1005Metric" (id 2) (at 77.1652 57.15 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 76.2 60.96 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 33b4648a-7ef4-4a94-849b-05a800800e7e))
+    (pin "2" (uuid 89f82548-d5d4-4642-ac88-127123fffcc8))
+  )
+
+  (symbol (lib_id "power:PWR_FLAG") (at 123.19 49.53 0) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 869591a6-d1fa-40fd-9371-e0082e3c12b8)
+    (property "Reference" "#FLG0103" (id 0) (at 123.19 47.625 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Value" "PWR_FLAG" (id 1) (at 123.19 44.45 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Footprint" "" (id 2) (at 123.19 49.53 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 123.19 49.53 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 4f9e86c7-4928-4fb5-a9a1-64c749717434))
+  )
+
+  (symbol (lib_id "power:VBUS") (at 54.61 116.84 0) (unit 1)
+    (in_bom yes) (on_board yes) (fields_autoplaced)
+    (uuid 8864208c-b15e-4a70-b251-0e977c94366c)
+    (property "Reference" "#PWR0102" (id 0) (at 54.61 120.65 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Value" "VBUS" (id 1) (at 54.61 111.76 0))
+    (property "Footprint" "" (id 2) (at 54.61 116.84 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "" (id 3) (at 54.61 116.84 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 9c78cf81-cb38-4e06-b355-94c55ae74a0b))
+  )
+
+  (symbol (lib_id "Device:LED") (at 67.31 40.64 180) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 886e3b1f-fbc6-4ec2-a4d0-d74356525a3f)
+    (property "Reference" "D1" (id 0) (at 67.31 35.56 0))
+    (property "Value" "LED" (id 1) (at 67.31 38.1 0))
```

Comment:
Do we want this LED? If its just going to be internal I think we can leave it off or have an unpopulated footprint we can use just for testing. If the latter, we should put the colour in the value field as this makes a lot of difference to the forward voltage.

Reply:
Replaced this with a 3.3V logic-compatible output in cd04ce8, so STAT can be used to drive the main indicator LED if desired. The voltage divider would not be required if we used the MCP73832T instead (which is in all other ways equivalent), but this is out of stock on JLCPCB.

Reply:
STAT is connected to MCU in d5d6c5ddbc67e39fed1ce54bdeec3217946e603a.